src/types/observablemap.ts

```diff
@@ -379,9 +377,7 @@ export class ObservableMap<K = any, V = any>
      * Note that the values migth still be observable. For a deep clone use mobx.toJS.
      */
     toJS(): Map<K, V> {
-        const res: Map<K, V> = new Map()
-        this._keys.forEach(key => res.set(key, this.get(key)!))
-        return res
+        return new Map(this)
```

Comment:
💯

Reply:
I think it relies on iteration protocol as well, will it work in IE?